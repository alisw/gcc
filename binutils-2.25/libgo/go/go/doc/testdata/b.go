../../../../../../libgo/go/go/doc/testdata/b.go