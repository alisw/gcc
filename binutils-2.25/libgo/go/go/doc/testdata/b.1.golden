../../../../../../libgo/go/go/doc/testdata/b.1.golden